src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -244,6 +271,21 @@ public boolean perform(final AbstractBuild<?, ? > build, final Launcher launcher
 
             reportStart(listener, subJob);
             List<Action> actions = new ArrayList<Action>();
+
+            if (resume) {
+                SubBuild subBuild = failedBuildMap.get(subJob.getUrl());
+                if (null != subBuild) {
+                    Item item = Jenkins.getInstance().getItem(subBuild.getJobName(), multiJobBuild.getParent(),

```

Comment:
Be aware about potential issues with folders. I'm not sure how MultiJob plugin works with them, but this code looks to be tricky